# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rav1e/hotfix-missing-rustdoc.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/Cargo.toml b/Cargo.toml
index 7c1ad5bd..4ef20009 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -131,7 +131,7 @@ features = ["png"]
 
 [build-dependencies]
 cc = { version = "1.0", optional = true, features = ["parallel"] }
-built = { version = "0.7.1", features = [] }
+built = { version = "0.7.2", features = [] }
 
 [build-dependencies.nasm-rs]
 version = "0.3"
